deque/test/test.mbt

```diff
@@ -197,10 +199,107 @@ test "iter2" {
 
 test "op_get" {
   let dv = @deque.of([1, 2, 3, 4, 5])
-  inspect([dv[0], dv[1], dv[2], dv[3], dv[4]], content="[1, 2, 3, 4, 5]")?
+  inspect(dv, content="Deque::[1, 2, 3, 4, 5]")?
   dv.pop_front() |> ignore
   dv.push_back(1)
   inspect(dv[4], content="1")?
   dv.push_front(2)
   inspect(dv[0], content="2")?
 }
+
+test "from_array" {
+  inspect(@deque.from_array(([] : Array[Int])), content="Deque::[]")?
+  inspect(@deque.Deque::[1, 2, 3], content="Deque::[1, 2, 3]")?
+}
+
+test "panic pop_front_exn" {
+  @deque.from_array(([] : Array[Int])).pop_front_exn()
+}
+
+test "panic pop_back_exn" {
+  @deque.from_array(([] : Array[Int])).pop_back_exn()
+}
+
+//Don't format heer!!! See Issue #632
+test "panic op_get" {
+  @deque.Deque::[1, 2, 3][-1] |> ignore
+}
```

Comment:
This bug needs to be investigated (cc @Yoorkin )
For this specific use case, we are planning to deprecate the syntactic sugar to encourage user only use `@deque.of([1,2,3])`